### ISSUE TITLE
How to use columnProcessor and RowProcessor (both at same time) for reading one file?

### ISSUE THREAD

**Issue by abhi18121** [uniVocity/univocity-parsers#206]

**Comment by jbax** [comment]
Just create your own `RowProcessor` implementation that wraps the implementations you want to use. For example:

```
public class CompositeProcessor implements RowProcessor {

	private final RowProcessor processors[];

	public CompositeProcessor(RowProcessor... processors) {
		this.processors = processors;
	}

	@Override
	public void processStarted(ParsingContext context) {
		for(RowProcessor processor : processors){
			processor.processStarted(context);
		}
	}

	@Override
	public void rowProcessed(String[] row, ParsingContext context) {
		for(RowProcessor processor : processors){
			processor.rowProcessed(row, context);
		}
	}

	@Override
	public void processEnded(ParsingContext context) {
		for(RowProcessor processor : processors){
			processor.processEnded(context);
		}
	}
}
```

Hope this helps.

**Comment by jbax** [comment]
Released version 2.5.9 with this enhancement (and others)
